CIScripts/Test/Tests/vRouterAgentService.Tests.ps1

```diff
@@ -24,12 +26,14 @@ Describe "vRouter Agent service" {
     
     Context "disabling" {
         It "is disabled" {
-            Assert-IsAgentServiceDisabled -Session $Session
+            Get-AgentServiceStatus -Session $Session `
+                | Should Be "Stopped"
         }
 
         It "does not restart" {
-            Start-Sleep -s 2
-            Assert-IsAgentServiceDisabled -Session $Session
+            {{ "Running" -eq (Get-AgentServiceStatus -Session $Session) } `
+                | Retry-UntilTrue -IntervalInSecs 0.2 -TimeoutInSecs 2 } `
+                | Should Throw
```

Comment:
IMHO these nested piped blocks are hard to understand. Perhaps consider passing them explicitely by `-ScriptBlock` or change indentation to be more obvious? It feels like we try shoe-horn this test into functions that we have (so definitely, `Consistently` would be nicer, even writing our own `Consistently` would be clearer too)

> `-IntervalInSecs 0.2`

Unfortunately, `Start-Sleep -Seconds` takes an int as a parameter, so `0.2` gets casted to int and rounded to `0`. If you want sub-second resolution of sleep, you have to use the `-Milliseconds` parameter.